### ISSUE TITLE
Uncaught TypeError: Bad argument

### ISSUE THREAD

**Issue by IronJaeger** [kriscross07/atom-gpp-compiler#25]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Microsoft Windows 10 Pro Insider Preview
**Thrown From**: [gpp-compiler](https://github.com/kriscross07/atom-gpp-compiler) package, v3.0.2

### Stack Trace

Uncaught TypeError: Bad argument

```
At internal/child_process.js:274

TypeError: Bad argument
    at TypeError (native)
    at ChildProcess.spawn (internal/child_process.js:274:26)
    at Object.exports.spawn (child_process.js:374:9)
    at compile (C:\Users\IronJaeger\.atom\packages\gpp-compiler\index.js:204:31)
    at tree-view-file.treeCompile (C:\Users\IronJaeger\.atom\packages\gpp-compiler\index.js:186:3)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\IronJaeger\AppData\Local\atom\app-1.8.0\resources\app.asar\src\command-registry.js:260:29)
    at CommandRegistry.handleCommandEvent (C:\Users\IronJaeger\AppData\Local\atom\app-1.8.0\resources\app.asar\src\command-registry.js:3:61)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (C:\Users\IronJaeger\AppData\Local\atom\app-1.8.0\resources\app.asar\src\command-registry.js:160:19)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (C:\Users\IronJaeger\AppData\Local\atom\app-1.8.0\resources\app.asar\src\atom-environment.js:1106:28)
    at EventEmitter.outerCallback (C:\Users\IronJaeger\AppData\Local\atom\app-1.8.0\resources\app.asar\src\application-delegate.js:340:25)
    at emitThree (events.js:97:13)
    at EventEmitter.emit (events.js:175:7)
```

### Commands

```
     -6:51.6.0 core:move-right (atom-text-editor.editor.is-focused)
     -6:50.9.0 editor:newline (atom-text-editor.editor.is-focused)
  2x -6:47.1.0 core:backspace (atom-text-editor.editor.is-focused)
     -6:25.9.0 core:undo (atom-text-editor.editor.is-focused)
  2x -6:23.8.0 editor:newline (atom-text-editor.editor.is-focused)
     -5:28.8.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:57.1.0 core:save (atom-text-editor.editor.is-focused)
     -0:53.9.0 core:copy (atom-text-editor.editor.is-focused)
     -0:47.3.0 core:save (atom-text-editor.editor.is-focused)
     -0:20.7.0 tree-view:reveal-active-file (li.texteditor.tab.sortable.active.right-clicked)
     -0:08.6.0 gpp-compiler:tree-compile (span.name.icon.icon-file-text)
```

### Config

```json
{}
```

### Installed Packages

```coffee
# User
gpp-compiler, v3.0.2

# Dev
No dev packages
```

**Comment by kriscross07** [comment]
1. delete the gpp-compiler package in Atom
2. `git clone` or download the [debug branch](../tree/debug)
3. open your terminal
4. `cd` into the downloaded directory
5. run `apm link`
6. reload Atom
7. press Ctrl + Shift + I (Linux) or Ctrl + Alt + i (Windows)
8. click on the `Console` tab
9. open your C / C++ file
10. press F5 to compile
11. reply with the output of Atom's console

**Comment by IronJaeger** [comment]
Solved by updating to version 3.0.4  , thanks
